app/controllers/application_controller.rb

```diff
@@ -7,21 +7,28 @@ class ApplicationController < ActionController::Base
   # Before Actions #############################################################
 
   def user_has_position(position)
-    unless current_user && current_user.position == position
-      flash[:notice] = "Sorry but you must be the #{position} to view that page"
+    unless current_user && !current_user.position.nil? &&
+           current_user.position == position
```

Comment:
Does this break down if there are multiple position objects that are all the same position. 2 people have the position or something